src/connectors/current-refinements/connectCurrentRefinements.js

```diff
@@ -0,0 +1,305 @@
+              createURL(clearRefinementFromState(helper.state, refinement)),
+            refinements,
+            instantSearchInstance,
+            widgetParams,
+          },
+          true
+        );
+      },
+
+      render({ results, helper, state, createURL, instantSearchInstance }) {
+        const refinements = transformItems(
+          getNormalizedRefinements({
+            results,
+            state,
+            helper,
+            includedAttributes,
+            excludedAttributes,
+            clearsQuery,
+          })
+        );
+
+        renderFn(
+          {
+            refine: refinement => clearRefinement(helper, refinement),
+            createURL: refinement =>
+              createURL(clearRefinementFromState(helper.state, refinement)),
+            refinements,
+            instantSearchInstance,
+            widgetParams,
+          },
+          false
+        );
+      },
+
+      dispose() {
+        unmountFn();
+      },
+    };
+  };
+}
+
+function getNormalizedRefinements({
+  results,
+  state,
+  helper,
+  includedAttributes,
+  excludedAttributes,
+  clearsQuery,
+}) {
+  const refinements = getRefinements(results, state, clearsQuery)
+    .filter(
+      refinement =>
+        !includedAttributes ||
+        includedAttributes.indexOf(refinement.attributeName) !== -1
+    )
+    .filter(
+      refinement => excludedAttributes.indexOf(refinement.attributeName) === -1
+    )
+    .map(normalizeRefinementItem);
+
+  return normalizeRefinements(refinements, helper);
+}
+
+function clearRefinementFromState(state, refinementItem) {
+  switch (refinementItem.type) {
+    case 'facet':
+      return state.removeFacetRefinement(
+        refinementItem.attribute,
+        refinementItem.value
+      );
+    case 'disjunctive':
+      return state.removeDisjunctiveFacetRefinement(
+        refinementItem.attribute,
+        refinementItem.value
+      );
+    case 'hierarchical':
+      return state.clearRefinements(refinementItem.attribute);
+    case 'exclude':
+      return state.removeExcludeRefinement(
+        refinementItem.attribute,
+        refinementItem.value
+      );
+    case 'numeric':
+      return state.removeNumericRefinement(
+        refinementItem.attribute,
+        refinementItem.operator,
+        refinementItem.value
+      );
+    case 'tag':
+      return state.removeTagRefinement(refinementItem.value);
+    case 'query':
+      return state.setQueryParameter('query', '');
+    default:
+      throw new Error(
+        `clearRefinement: type ${refinementItem.type} is not handled`
+      );
+  }
+}
+
+function clearRefinement(helper, refinement) {
+  helper.setState(clearRefinementFromState(helper.state, refinement)).search();
+}
+
+function getOperatorSymbol(operator) {
+  switch (operator) {
+    case '>=':
+      return '≥';
+    case '<=':
+      return '≤';
+    default:
+      return '';
+  }
+}
+
+function normalizeRefinementItem(item) {
```

Comment:
Could be done in https://github.com/algolia/instantsearch.js/blob/5484c62e2fa29938305718efd82f0a585c8dde30/src/lib/utils.js#L227 but it'd be a huge refactor.